baselines/dom.generated.d.ts

```diff
@@ -11081,28 +11122,60 @@ declare var Notification: {
 interface OES_element_index_uint {
 }
 
+declare var OES_element_index_uint: {
```

Comment:
Hmm, I suppose we would first need #873 to land? These now have `[LegacyNoInterfaceObject]` in their IDL...

Reply:
#873 is now merged.